internal/k8s/ako_init.go

```diff
@@ -642,31 +651,64 @@ func (c *AviController) DeleteModels() {
 	}
 }
 
-func SyncFromIngestionLayer(key string, wg *sync.WaitGroup) error {
+func SyncFromIngestionLayer(key interface{}, wg *sync.WaitGroup) error {
 	// This method will do all necessary graph calculations on the Graph Layer
 	// Let's route the key to the graph layer.
 	// NOTE: There's no error propagation from the graph layer back to the workerqueue. We will evaluate
 	// This condition in the future and visit as needed. But right now, there's no necessity for it.
 	//sharedQueue := SharedWorkQueueWrappers().GetQueueByName(queue.GraphLayer)
-	nodes.DequeueIngestion(key, false)
+
+	keyStr, ok := key.(string)
+	if !ok {
```

Comment:
If the key isn't a string then what can it be?
If so - does it point to a problem in the previous layer?
Also - if it does occur, and we return nil in Line 663 - looks like we won't be able to debug as to why a certain worker couldn't process anything, so putting a log statement would aid in that.